### ISSUE TITLE
The reason of setting specific calibration resistor

### ISSUE THREAD

**Issue by itkwon** [WuMRC/advanced-imaging#1]
Dear @tunjid ,

I want to ask why you choose the calibration resistance 461 ohm on Multiplexer_Z program. 

If this choice based on the recent trend of the impedance value, I might redesign to select the different value of feedback resistor. Currently, I choose 1.84k Ohm as a feedback resistor.
Also, I choosed 332 Ohm as a feedback resistor to measure more than 450 Ohm resistance. If you estimate lower Bio-Impedance range, I need to lower the feedback resistance next time.

Thanks.

**Comment by barrybelmont** [comment]
Hello Il Taek. 

I implemented the Muliplexer_Z program based on previous work of TJ's. The calibration resistor in this case was mostly chosen at random and simply the one on hand. I believe I wrote/implemented this before you changed the default circuit to include a 332 ohm resistor. If you think the 332 ohm resistor would be better, try it and report back.

**Comment by itkwon** [comment]
Thank you so much @barrybelmont .
Actually, I discovered that I make some mistakes on my previous post. In other words, I mixed two questions into one. 
1. I mixed up the feedback resistor and calibration resistor.
2. I wanted to ask why he used other different feedback & calibration resistor.
3. I also wanted to ask if the estimated range lowered to change my design.

However, you answer all question I raised, so thanks a lot!
